tools/build-fs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+
+# To use this script, run build-fs test or build-fs main
+# If no network is specified, or a different network is specified, the env's 
+#   version of the following variables are used
+#   - SGX_MODE
+#   - IAS_MODE
+#   - CONSENSUS_ENCLAVE_CSS
+#   - NAMESPACE (if no namespace is present, we use 'default')
```

Comment:
I don't see the string `default` mentioned anywhere in this script other than this comment.
On this topic, what do we get from allowing this script to run without a namespace?